timemachine/fe/single_topology.py

```diff
@@ -166,6 +178,19 @@ def setup_dummy_interactions(
         if all([a in dga for a in idxs]):
             dummy_improper_idxs.append(tuple([int(x) for x in idxs]))
             dummy_improper_params.append(params)
+    for idxs, params in zip(chiral_atom_idxs, chiral_atom_params):
+        # consider a few cases: (1) all idxs dummy, (2) a single root_anchor_atom and 3 dummies, (3) other
+        center, i, j, k = idxs
+        case_1 = all(a in dummy_group for a in (center, i, j, k))
+        three_in_dummy_group = sum(a in dummy_group for a in (center, i, j, k)) == 3
+        one_is_root_anchor_atom = sum(a == root_anchor_atom for a in (center, i, j, k)) == 1
```

Comment:
@maxentile  sanity check this?

Reply:
Looks safe: there's no way for a potential function of this form to influence the distribution of `root_anchor_atom`.

To test the TODO cases below, I'll do a little numerical check.

(Define 4-atom system. Fix `root_anchor_atom` at origin. Add some arbitrary (not necessarily radially symmetric) energy function in terms of position of `neighbor_anchor_atom`. Add the 4-body energy term to be tested. Assert distribution of `neighbor_anchor_atom`'s position is unaffected by changing placement of remaining two atoms.)